Tasks/BuildctlV0/task.loc.json

```diff
@@ -0,0 +1,61 @@
+{
+  "id": "8413C881-4959-43D5-8840-B4EA0FFC5CFE",
+  "name": "BuildKitInstaller",
+  "friendlyName": "ms-resource:loc.friendlyName",
+  "description": "ms-resource:loc.description",
+  "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks",
+  "helpMarkDown": "ms-resource:loc.helpMarkDown",
+  "category": "Tool",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 160,
+    "Patch": 1
+  },
+  "demands": [],
+  "satisfies": [
+    "Buildctl"
+  ],
+  "groups": [],
+  "inputs": [
```

Comment:
The task interface is not in alignment with Docker@2. Following inputs need to be added - 
- dockerRegistryServiceConnection (In Docker@2, it's called 'containerRegistry', but based on feedback let's explicitly call it 'dockerRegistryServiceConnection'
- repository
- tags
- arguments
- addPipelineMetadata (it's called addPipelineData in Docker@2. Small change to addPipelineMetadata here)

Reply:
added these parameters for now... still not using using it as push functionlity is yet to be added